app/src/main/kotlin/org/eclipse/kuksa/companion/feature/navigation/view/AdaptiveNavigationView.kt

```diff
@@ -50,6 +52,18 @@ fun AdaptiveNavigationView(
     }
 }
 
+class AdaptiveNavigationView private constructor() {
+    companion object {
+        fun calculatePaddingValues(windowSizeClass: WindowSizeClass): PaddingValues {
```

Comment:
I basically wanted to access these by AdaptiveNavigationView.PaddingValues instead of adding a top level "adaptiveNavigationViewPaddingValues...()" method, which is always autocompleted. 

Maybe you can think of a better way. I also intentionally decided against a WindowSizeClass-Extension.